enos/modules/run_workloads/outputs.tf

```diff
@@ -1,16 +1,27 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: BUSL-1.1
 
-/* output "jobs_count" {
-  value = length(local.job_names)
-} */
-
 output "jobs_count" {
   description = "The number of jobs thar should be running in the cluster"
+  value       = length(var.workloads) + chomp(enos_local_exec.get_jobs.stdout)
+}
+
+output "new_jobs_count" {
+  description = "The number of jobs that were triggered by the module"
   value       = length(var.workloads)
 }
 
 output "allocs_count" {
   description = "The number of allocs that should be running in the cluster"
-  value       = sum([for wl in var.workloads : wl.alloc_count])
+  value       = local.system_job_count * chomp(enos_local_exec.get_nodes.stdout) + local.service_batch_allocs + chomp(enos_local_exec.get_allocs.stdout)
+}
+
+output "nodes" {
+  description = "Number of current clients in the cluster"
+  value       = chomp(enos_local_exec.get_nodes.stdout)
+}
+
+output "new_allocs_count" {
+  description = "The number of allocs that should be running in the cluster"
+  value       = local.system_job_count * chomp(enos_local_exec.get_nodes.stdout) + local.service_batch_allocs
 }
```

Comment:
What's this output for? It doesn't match `allocs_count`

Reply:
Im trying to make this module aware of existing allocs so it outputs all the running allocs, new and old, and the output can be directly used for a next step in enos, because it does not accept functions as step.variables, so this output helps me debug

Reply:
Makes sense! Let's update the descriptions to make that clear. Right now this is the same as `allocs_count`.